src/debug/di/arm64/floatconversion.S

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <unixasmmacros.inc>
+
+// Arguments
+//     input: (in R0) the adress of the ULONGLONG to be converted to a double
```

Comment:
Just to be consistent with the rest of our code, should we make this: 

```suggestion
//     input: (in X0) the address of the _NEON128 to be converted to a double
```

Although the manual does call it R0.